hrui/js/src/civihr-popup/civihr-popup.js

```diff
@@ -12,6 +12,11 @@ Prevent Popups to overflow tables.
     if ($button.hasClass('btn-slide-active')) {
       $popup = $button.children('ul.panel');
 
+      // Use id of parent tr tag
```

Comment:
Both "Use id of parent tr tag", "Use class of parent tr tag" are pretty meaningless comments (I can already see what you are doing, but you're not telling me *why*).

Given that the reason why you do this is a bit complicated to explain in a comment, just leave the number of the ticket

```js
// PCHR-1784
$popup.attr(/* .. */);
$popup.addClass(/* .. */);